### PR TITLE
Updated TooltipBestPractices to include new best practices

### LINKED ISSUE #17562: Missing accessibility best practices information for Tooltip on non-focusable UI
### Environment Information

- **Package version(s)**: 7.146
- **Browser and OS versions**: any

### Describe the issue:

The documentation for [Tooltip](https://developer.microsoft.com/en-us/fluentui#/controls/web/tooltip) should be updated for how to handle cases where a tooltip is desired around some UI that is not itself focusable.

(Edit - re-worded this as 'not itself *focusable*' vs interactive, since it's specifically the lack of keyboard focus that's the issue here.)

A good example of this is Tooltip only on overflow example on that page. When the text is made to truncate, mouse users can access the full text via the tooltip, but keyboard users cannot, as there is no element to tab to. (Screenreader users may or may not be able to access the content, depending on whether the content in the DOM is merely truncated via CSS for display, but this may not work with other types of truncation or if the tooltip is conveying information that's not in the DOM.)

Adding `tabindex=0` might help somewhat; but this now results in an element that is interactive, but without a clearly defined ARIA `role`. Adding `role=button` would be inappropriate here, as the UI element is not actually executing a command or acting like a button; it is merely focusable in order that focus has somewhere to dwell to cause the tooltip to appear.

While it may tempting to discourage the use of Tooltip around non-interactive items outright, it does appear that in reality there are occasionally cases where this is needed, so some robust best practice should be investigated and presented, and supported by the control itself if appropriate.

This issue likely also occurs where tooltips are placed around disabled buttons to explain why the button is disabled; disabled controls are generally not focusable, so again keyboard users have no way to activate the tooltip.

### PR DESCRIPTION
**Pull request by tabithahsia**
## Previous Behavior
Tooltip read me best practices did not have guidelines on non-interactive elements.

## New Behavior

Tooltip read me best practices now has guidelines on non-interactive elements.

## Related Issue(s)

- Fixes #17562

### CODE REVIEW

File: packages/react-examples/src/react/Tooltip/docs/TooltipBestPractices.md [hunk 1]
```diff
@@ -16,3 +16,7 @@ For a truncated label or a label that’s likely to truncate in some langua
 - Provide the untruncated label in the tooltip.
 - Don't provide a tooltip if the untruncated info is provided elsewhere on the page or flow.
 - Optional: On another line, provide a clarifying description, but only if needed.
+
+### Accessibility
+
+- Don't add tooltips to unfocusable or disabled controls. Screen readers can't consistently access or read tooltips associated with elements that can't receive focus or are disabled. Consider having the information statically available on the page or through a "help"-style toggle button.
```
Comment (smhigley) [inline packages/react-examples/src/react/Tooltip/docs/TooltipBestPractices.md:22]:
```suggestion
- Don't add tooltips to unfocusable or disabled controls. People using the keyboard or screen readers can't consistently access or read tooltips associated with elements that can't receive focus or are disabled. Consider having the information statically available on the page or through a "help"-style toggle button.
```

Just a suggestion to include keyboard users, and add "people" ("users" would work too) as the subject

**Review by smhigley — COMMENTED** [review]
Just the one suggestion, otherwise it LGTM!

**Review by smhigley — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5b6536c989c9ebb2bf4010fbf18dfb50b570b2c6'>5b6536c989c9ebb2bf4010fbf18dfb50b570b2c6</a></sub>
<div id="bundleSizeComment9423"/>

**Review by khmakoto — APPROVED** [review]

**Pull request merged**